### ISSUE TITLE
The tags have disappeared and a branch is missing

### ISSUE THREAD

**Issue by jean-christophe-manciot** [htacg/tidy-html5#465]
Last tag is 5.1.25 on master and 5.2.0 on release/5.2 with many 5.3.x bumps since then.
``` 
git checkout master
git describe --tags
5.1.25-220-g4edbc24

git checkout release/5.2
git describe --tags
5.2.0-2-g1a34203
```
It seems also that the branch release/5.3 is missing:
```
git branch --all
* (HEAD detached at 4edbc24)
  master
  remotes/origin/329_rebased
  remotes/origin/329_rebased_merged
  remotes/origin/HEAD -> origin/master
  remotes/origin/attrdict_phase3
  remotes/origin/issue-329
  remotes/origin/issue-380
  remotes/origin/issue-428
  remotes/origin/master
  remotes/origin/poconvert_enhance
  remotes/origin/release/5.0.0
  remotes/origin/release/5.2
```

**Comment by geoffmcl** [comment]
@jean-christophe-manciot thanks for your comment...

Maybe [VERSION.md](https://github.com/htacg/tidy-html5/blob/master/README/VERSION.md) is a little unclear, but we use odd/even `minor` version numbering, where only `even` number indicates a release, and tags are **only** created on a release...

So to date we have had only two relatively recent releases, 5.0, and 5.2, and the next 5.4, has not been done. Oh, and yes, there was an exception, 5.1.25, along the way, which I hope will not be repeated...

So, sort of nothing is missing - tags are only added for releases, and there have been none since 5.2...

While there have been quite a number of **fixes**, and **features** added since 5.2, there also seems a considerable number of outstanding, important issues, which need attention... any, all help appreciated...

But agree it is about time for another release... and consider `master` 5.3.12 a good, stable, candidate... will try to keep the present Dec 15, 2016 schedule for 5.4... 

What do others think? Thanks...

**Comment by balthisar** [comment]
Since the actual issue raised has been addressed, I will close this.
